pages/account.vue

```diff
@@ -261,14 +246,15 @@ async function handleLikerPlusButtonClick() {
   }
 }
 
-async function handleCustomerServiceChatButtonClick() {
-  useLogEvent('customer_service', { method: 'chat' })
-  if (crisp && isCrispLoaded.value) {
-    crisp.do('chat:open')
+async function handleCustomerServiceLinkButtonClick() {
+  if (!isCrispLoaded.value) {
```

Comment:
does `@click` event trigger even `:disabled="!isCrispLoaded"`?

Reply:
@AuroraHuang22 button is hidden when Crisp is not loaded
<img width="651" alt="Screenshot 2025-06-19 at 09 36 57" src="https://github.com/user-attachments/assets/5f3a7cf1-776c-4611-b472-ef1ed941a699" />